lib/output.rb

```diff
@@ -4,35 +4,44 @@
 class Output
   attr_reader :tcp_server,
               :client,
-              :request_lines
+              :request_lines,
+              :output,
+              :counter
   def initialize
-    @tcp_server = TCPServer.new(9292)
-    @client = tcp_server.accept
     @request_lines = []
+    @counter = 0
   end
 
-  def make_request
-    puts "Ready for a request"
-    while line = @client.gets and !line.chomp.empty?
-      @request_lines << line.chomp
-    end
+  def increment_counter
+    @counter += 1
+  end
```

Comment:
Might want to consider renaming @counter and think about whether it is a state of `Output`